posthog/queries/actor_base_query.py

```diff
@@ -184,6 +184,9 @@ def get_groups(
 def get_people(team_id: int, people_ids: List[Any]) -> Tuple[QuerySet[Person], List[SerializedPerson]]:
     """ Get people from raw SQL results in data model and dict formats """
     persons: QuerySet[Person] = Person.objects.filter(team_id=team_id, uuid__in=people_ids)
```

Comment:
Any reason to split these into three?

```suggestion
    persons: QuerySet[Person] = Person.objects.filter(team_id=team_id, uuid__in=people_ids).prefetch_related(....).only(...)
```

Reply:
Copy + paste. Fixed!